src/Servers/IIS/AspNetCoreModuleV2/RequestHandlerLib/requesthandler_config.cpp

```diff
@@ -101,6 +102,8 @@ REQUESTHANDLER_CONFIG::Populate(
     BSTR                            bstrBasicAuthSection = NULL;
     BSTR                            bstrAnonymousAuthSection = NULL;
     BSTR                            bstrAspNetCoreSection = NULL;
+    auto launcherPathEnv = Environment::GetEnvironmentVariableValue(CS_ANCM_LAUNCHER_PATH);
+    auto launcherArgsEnv = Environment::GetEnvironmentVariableValue(CS_ANCM_LAUNCHER_ARGS);
```

Comment:
We need to be careful about calling GetEnvironmentVariableValue in the out of proc code. For example, if GetEnvironmentVariableValue fails, it throws an exception. I'd recommend try/catching around these call or checking where the next try catch is.

Reply:
Updated